packages/server/lib/controllers/sync.controller.ts

```diff
@@ -386,6 +391,11 @@ class SyncController {
                 operation_name: action_name
             };
 
+            span.setTag('actionName', action_name)
+                .setTag('connectionId', connectionId)
```

Comment:
I wonder if we should prefix all our tags with `nango.` to differentiate them from plugins or built-in tags. what do you think @khaliqgant @bodinsamuel ?

Reply:
yep agreed